src/main/kotlin/com/ampnet/walletservice/grpc/blockchain/BlockchainService.kt

```diff
@@ -31,4 +32,5 @@ interface BlockchainService {
     fun generateTransferPlatformManager(address: String, coop: String): TransactionData
     fun getSellOffers(coop: String): List<SellOfferData>
     fun deployCoopContract(coop: String, address: String)
+    fun getUserWalletsWithInvestment(coop: String): List<UserWalletsForCoopAndTxTypeResponse.WalletWithHash>
```

Comment:
Import sub-class `WalletWithHash` or create a type alias that is shorter than `UserWalletsForCoopAndTxTypeResponse.WalletWithHash`